tools/ExtensionLibrary/DevHome.ExtensionLibrary/ViewModels/ExtensionLibraryViewModel.cs

```diff
@@ -69,6 +66,21 @@ public async Task LoadedAsync()
     {
         await GetInstalledPackagesAndExtensionsAsync();
         GetAvailablePackages();
+
+        if (_extensionService != null)
+        {
+            _extensionService.OnExtensionsChanged -= OnExtensionsChanged;
+            _extensionService.OnExtensionsChanged += OnExtensionsChanged;

```

Comment:
Why do we do this at all? To trigger some kind of "on subscribe" event?

Reply:
So we can update the page when the list changes -- for example, if something shows up in the list of extensions available in the store, and then you install it, you'll see the UI update it from being in that list to the list of installed extensions.